onnxruntime/test/providers/cpu/tensor/resize_op_test.cc

```diff
@@ -2287,6 +2289,106 @@ TEST(ResizeOpTest, Antialias_NHWCBicubic_ExcludeOutside) {
   TestAntialiasing({{"mode", "cubic"}, {"exclude_outside", "0"}}, {1, 4, 6, 2}, X, {1, 8, 4, 2}, Y, excluded_eps);
 }
 
+TEST(ResizeOpTest, NoAntialias_AlignCorners_Cubic_Floor_NCHW) {
+  std::vector<float> X(24);
+  std::iota(X.begin(), X.end(), 1.0f);
+  std::vector<float> Y = {
```

Comment:
nip: Are these data generated with a scripts? I usually attach the script to the test so that if someone want to add more test for different shape, data type or attributes, or for a new version of the op, one does not need to recreate a script.

Reply:
Added [onnxruntime/test/providers/cpu/tensor/resize_sample_test_gen.py](https://github.com/microsoft/onnxruntime/pull/24385/files#diff-1b0a6e5459c52f16938d3c8bbd6e76102b6be204db8e43a065d37c04eddd0045)